src/commands/createFunction/durableSteps/netherite/NetheriteConfigureHostStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '@microsoft/vscode-azext-azureappservice/out/src/extensionVariables';
+import { AzExtFsExtra, AzureWizardExecuteStep, nonNullValue } from '@microsoft/vscode-azext-utils';
+import * as path from 'path';
+import { hostFileName } from '../../../../constants';
+import { hostJsonConfigFailed } from '../../../../constants-nls';
+import { IHostJsonV2, INetheriteTaskJson } from '../../../../funcConfig/host';
+import { netheriteUtils } from '../../../../utils/durableUtils';
+import { IEventHubsConnectionWizardContext } from '../../../appSettings/IEventHubsConnectionWizardContext';
+
+export class NetheriteConfigureHostStep<T extends IEventHubsConnectionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 250;
```

Comment:
This has the same priority as `OpenFolderStep`. We should avoid using the same priority number for execute steps. I'm not sure if they'd ever be used in the same wizard, but I think it's possible if you create a new project and your first function trigger is durable.

Even if it's not possible, it's probably still better practice to avoid conflicts. You can change this one to 245 or something, or increase `OpenFolderStep` to 300.